### ISSUE TITLE
Webpack hangs when building - Windows

### ISSUE THREAD

**Issue by dearruma** [workco/marvin#33]
Webpack hangs when running "npm run build" in windows. I have modified the script with 
"rimraf build && SET NODE_ENV=production && webpack"

It hangs while building module.js.

Please help!

**Comment by Stanko** [comment]
ATM I have no windows machine to test it on. Can you please paste error logs?

**Comment by RandomClown0** [comment]
I get:

```
C:\xampp\htdocs\CTS\marvin-master>npm run build

> react-redux-webpack2-boilerplate@0.1.7 build C:\xampp\htdocs\CTS\marvin-master
> rm -rf ./build && NODE_ENV="production" webpack

'NODE_ENV' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\System Tools\\Development\\Node\\node.exe" "C:\\System Tools\\Development\\Node\\node_modules\\npm\\bin\\npm-cli.js" "run" "build"
npm ERR! node v7.10.0
npm ERR! npm  v4.2.0
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-redux-webpack2-boilerplate@0.1.7 build: `rm -rf ./build && NODE_ENV="production" webpack`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-redux-webpack2-boilerplate@0.1.7 build script 'rm -rf ./build && NODE_ENV="production" webpack'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-redux-webpack2-boilerplate package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rm -rf ./build && NODE_ENV="production" webpack
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-redux-webpack2-boilerplate
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-redux-webpack2-boilerplate
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\edward.thang\AppData\Roaming\npm-cache\_logs\2017-07-27T18_03_57_234Z-debug.log
```

**Comment by RandomClown0** [comment]
***(edit misread op)***

Just tried the command & worked for me.

How long was it hanging? It took a while (like 30s) for Webpack to output on my end.

**Comment by Stanko** [comment]
Closing this as we released v1.0.0 and this is probably outdated. Please let me know if error still occurs.
